stdlib/2and3/distutils/ccompiler.pyi

```diff
@@ -19,6 +19,16 @@ def new_compiler(plat: Optional[str] = ..., compiler: Optional[str] = ...,
 def show_compilers() -> None: ...
 
 class CCompiler:
+    dry_run: int
+    force: int
+    verbose: int
+    output_dir: str
+    macros: List[_Macro]
```

Comment:
`_Macro` is misdefined above. The second element should be `Optional[str]`, not just `str`. Out of scope for this PR, but while you are at it ...

Reply:
good catch, fixed that as well